spec/active_record_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'spec_helper'
+require 'support/active_record/foo'
+require 'nulldb_rspec'
+
+ActiveRecord::Base.establish_connection(
+  adapter: :nulldb,
+  schema: 'spec/support/active_record/schema.rb'
+)
+
+NullDB.configure { |ndb| def ndb.project_root; Dir.pwd; end; }
+
+shared_examples 'produces_correct_sql' do 
+  it 'produces correct sql for first page' do
+    paginated_sql = ApiPagination.paginate(collection, per_page: per_page)
+    .to_sql
+    expect(paginated_sql).to eql(Foo.limit(per_page).offset(0).to_sql)
+  end
+end
+
+describe 'ActiveRecord Support' do 
+  let(:collection) { Foo.all }
+  let(:per_page) { 5 }
+
+  if ApiPagination.config.paginator == :kaminari
+    context 'pagination with kaminari' do
+      before { ApiPagination.config.paginator = :kaminari }
```

Comment:
We should be able to get rid of this `before` proc since the paginator is already `:kaminari`